packages/node-addon-examples/scripts/verify-prebuilds.mts

```diff
@@ -41,6 +44,9 @@ async function verifyApplePrebuild(dirent: fs.Dirent) {
   console.log("Verifying Apple prebuild", dirent.name, "in", dirent.parentPath);
   for (const arch of EXPECTED_XCFRAMEWORK_PLATFORMS) {
     const archDir = path.join(dirent.parentPath, dirent.name, arch);
+    if (!fs.existsSync(archDir)) {
+      return;
+    }
```

Comment:
This breaks an invariant and shouldn't fail silently.

Reply:
https://github.com/callstackincubator/react-native-node-api/pull/204#issuecomment-3117912710

Reply:
What would you prefer to be done?